cmd/promtool/main.go

```diff
@@ -135,6 +135,7 @@ func main() {
 
 	importCmd := tsdbCmd.Command("create-blocks-from", "[Experimental] Import samples from input and produce TSDB blocks. Please refer to the storage docs for more details.")
 	openMetricsImportCmd := importCmd.Command("openmetrics", "Import samples from OpenMetrics input and produce TSDB blocks. Please refer to the storage docs for more details.")
+	importHumanReadable := openMetricsImportCmd.Flag("human-readable", "Print human readable values.").Short('r').Bool()
```

Comment:
I think this flag will apply to all subcommands of `create-blocks-from`. Is it possible to make this a flag of `importCmd`?